draft-irtf-cfrg-frost.md

```diff
@@ -416,38 +417,47 @@ during the FROST protocol. The following function encodes a list of participant
 commitments into a byte string for use in the FROST protocol.
 
 ~~~
-  Inputs:
-  - commitment_list = [(i, hiding_nonce_commitment_i, binding_nonce_commitment_i), ...],
-    a list of commitments issued by each participant, where each element in the list
-    indicates a NonZeroScalar identifier i and two commitment Element values
-    (hiding_nonce_commitment_i, binding_nonce_commitment_i). This list MUST be sorted
-    in ascending order by identifier.
-
-  Outputs:
-  - encoded_group_commitment, the serialized representation of commitment_list, a byte string.
-
-  def encode_group_commitment_list(commitment_list):
-    encoded_group_commitment = nil
-    for (identifier, hiding_nonce_commitment, binding_nonce_commitment) in commitment_list:
-      encoded_commitment = G.SerializeScalar(identifier) ||
-                           G.SerializeElement(hiding_nonce_commitment) ||
-                           G.SerializeElement(binding_nonce_commitment)
-      encoded_group_commitment = encoded_group_commitment || encoded_commitment
-    return encoded_group_commitment
+Inputs:
+- commitment_list = [(i, hiding_nonce_commitment_i,
+  binding_nonce_commitment_i), ...], a list of commitments issued by
+  each participant, where each element in the list indicates a
+  NonZeroScalar identifier i and two commitment Element values
+  (hiding_nonce_commitment_i, binding_nonce_commitment_i). This list
+  MUST be sorted in ascending order by identifier.
+
+Outputs:
+- encoded_group_commitment, the serialized representation of
+  commitment_list, a byte string.
+
+def encode_group_commitment_list(commitment_list):
+  encoded_group_commitment = nil
+  for (identifier, hiding_nonce_commitment,
+      binding_nonce_commitment) in commitment_list:
+  encoded_commitment = (
+      G.SerializeScalar(identifier) ||
+      G.SerializeElement(hiding_nonce_commitment) ||
+      G.SerializeElement(binding_nonce_commitment)
+  )
+  encoded_group_commitment = (
+      encoded_group_commitment ||
+      encoded_commitment
+  )
```

Comment:
This seems like it should be indented underneath the `for`.

Reply:
Oops, thanks. Fixed in b376ec0